gravitee-kubernetes-client/src/test/java/io/gravitee/kubernetes/client/config/KubernetesConfigTest.java

```diff
@@ -29,18 +29,6 @@
 @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 class KubernetesConfigTest {
 
-    @Test
-    void should_load_default() {
-        KubernetesConfig instance = KubernetesConfig.getInstance();
-        assertThat(instance.getApiTimeout()).isPositive();
-        assertThat(instance.getWebsocketTimeout()).isPositive();
-        assertThat(instance.getApiVersion()).isEqualTo("v1");
-        assertThat(instance.verifyHost()).isTrue();
-        assertThat(instance.useSSL()).isTrue();
-
-        assertThat(instance).isSameAs(KubernetesConfig.getInstance());
-    }
```

Comment:
Why did you delete this test ?

Reply:
It is using a shared instance of configuration. Ideally I'd need execute this test first before all else. As you already know I can't do that. Other tests are modifying this instance, so there is no way I can have a stable build. I already test default value in another test. I just wanted to isolate it, but yeah... so static.